packages/plugins/blocks/blocks-antd/src/blocks/PageHeaderMenu/PageHeaderMenu.js

```diff
@@ -118,39 +128,26 @@ const PageHeaderMenu = ({
                 content: () => (
                   <>
                     <Link home={true}>
-                      <img
-                        src={
-                          (properties.logo && properties.logo.src) ||
-                          (get(properties, 'header.theme') === 'light'
-                            ? `${basePath}/logo-light-theme.png`
-                            : `${basePath}/logo-dark-theme.png`)
-                        }
-                        srcSet={
-                          (properties.logo && (properties.logo.srcSet || properties.logo.src)) ||
-                          (get(properties, 'header.theme') === 'light'
-                            ? `${basePath}/logo-square-light-theme.png 40w, ${basePath}/logo-light-theme.png 577w`
-                            : `${basePath}/logo-square-dark-theme.png 40w, ${basePath}/logo-dark-theme.png 577w`)
-                        }
-                        sizes={
-                          (properties.logo && properties.logo.sizes) ||
-                          '(max-width: 576px) 40px, 577px'
-                        }
-                        alt={(properties.logo && properties.logo.alt) || 'Lowdefy'}
-                        className={methods.makeCssClass([
-                          {
-                            width: 130,
-                            sm: {
-                              width:
-                                properties.logo && properties.logo.src && !properties.logo.srcSet
-                                  ? 130
-                                  : 40,
-                            },
-                            md: { width: 130 },
-                          },
-                          styles.logo,
-                          properties.logo && properties.logo.style,
-                        ])}
-                      />
+                      <picture>
+                        <source
+                          media={`(min-width:${properties.logo?.breakpoint ?? 577}px)`}
+                          srcSet={
+                            properties.logo?.src ??
+                            `${basePath}/logo-${properties.header?.theme ?? 'dark'}-theme.png`
+                          }
+                        />
+                        <img
```

Comment:
Shouldn't this the be a `next/image`?
https://nextjs.org/docs/api-reference/next/image